modules/apps/27-interchain-accounts/controller/keeper/relay.go

```diff
@@ -10,9 +10,12 @@ import (
 	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
 )
 
-// TrySendTx takes in a transaction from an authentication module and attempts to send the packet
-// if the base application has the capability to send on the provided portID
-func (k Keeper) TrySendTx(ctx sdk.Context, chanCap *capabilitytypes.Capability, portID string, icaPacketData icatypes.InterchainAccountPacketData) (uint64, error) {
+// TrySendTx takes pre-built packet data containing messages to be executed on the host chain from an authentication module and attempts to send the packet.
+// The packet sequence for the outgoing packet is returned as a result.
+// If the base application has the capability to send on the provided portID. An appropriate
+// absolute timeoutTimestamp must be provided. If the packet is timed out, the channel will be closed.
```

Comment:
Are there any pros or cons in using an absolute timestamp instead of relative? I am asking because for the transfer cli we have a relative timestamp, right? Maybe it's good to be consistent (if possible) on using either relative or absolute, but not a mix.